### ISSUE TITLE
Improvements for Files display on Item detail view page

### ISSUE THREAD

**Issue by murny** [ualbertalib/jupiter#447]
Files should show their name, type and size on the item view screen?
Currently for a single file it looks like:
![image](https://user-images.githubusercontent.com/1930474/35989226-d95f3ca4-0cbd-11e8-8779-5578bec7cfed.png)

and multiple file looks like:
![image](https://user-images.githubusercontent.com/1930474/35989115-981c89d6-0cbd-11e8-96bc-256e74d741ba.png)

Should be kinda like Deposit Review screen where at a glance you know exactly what the file looks like without having to download it:
![image](https://user-images.githubusercontent.com/1930474/35989079-7d602fda-0cbd-11e8-9adf-ac71e300ed55.png)

Would hate for someone to accidentally download a 1gb file on their iphone over their data network because you have no idea how big a file is

**Comment by weiweishi** [comment]
@sfbetz and @leahvanderjagt Please review this ticket, and comment what are your thoughts on this. Thank you.

**Comment by sfbetz** [comment]
Makes sense to me. Let's include the info as displayed in the "review and update item" screen just above the view download buttons. eg filename.pdf 613KB

**Comment by pbinkley** [comment]
Note that currently very long filenames overrun the left column and push into the main column, e.g. https://era-beta.library.ualberta.ca/items/6ede136f-683e-4e68-a403-c55d39fb0412

![image](https://user-images.githubusercontent.com/272816/37543931-db09a7ce-2928-11e8-808d-a37a1a99de6e.png)

**Comment by cwant** [comment]
Interesting, I don't get that behaviour for that item (in fact, I had added some CSS to prevent that from happening while addressing #401). Maybe a browser specific thing? Here is what I get:

![files-long-name](https://user-images.githubusercontent.com/672104/37544168-b3daa33c-2929-11e8-964f-38fd329613bb.png)

**Comment by pbinkley** [comment]
I get it on Firefox, but I just checked Chrome and I don't get it there (both on Ubuntu). Sorry for not crosschecking before I posted. Hopefully it's a small css tweak to bring Firefox into line.

**Comment by cwant** [comment]
Changing this from `break-word` to `break-all` fixes it on firefox for me:

https://github.com/ualbertalib/jupiter/blob/master/app/assets/stylesheets/items.scss#L11

I'll leave it to whomever takes this ticket to follow up.

**Comment by weiweishi** [comment]
Break this into two issues:
- The one with very long filenames overrun the left column and push into the main column in Firefox: #710
and 
- Include the filesize on file download #447

#447 requires introducing new predicates into Fedora, and a full data index, which will not likely to be required if our tech stack is going to be changed in the near future.  Also some design decision to be made. @sfbetz @leahvanderjagt based on our discussion the other day re: priority of this issue, I will move this into future enhancement. Please let me know if this concerns you.

**Comment by weiweishi** [comment]
now the information is available through ActiveStorage. We just need design for single file view to display the file level metadata. @weiweishi will follow up with @leahvanderjagt  on the design.
